### ISSUE TITLE
MaterialTopTabNavigator swipe gesture blocked by drawer from opposite side.

### ISSUE THREAD

**Issue by charles-goode** [react-navigation/react-navigation#7321]
### Current behaviour
I have three tabs and the left tab has a left side drawer navigator with `edgewidth` equal to the full width of the screen. Similar to react-navigation/tabs#134 , the swipe-able tabbar is no longer responding. I only opened a separate issue because mine is a drawerNavigator inside of a tabNavigator instead of the other way around (as it is in the other issue).  When I lower the edgewidth of the drawer it works fine, but I believe it should properly handle either gesture since they propogate in opposite directions (left-to-right) vs (right-to-left) originating from anywhere along the x-axis.  I should note I'm testing these changes only on Android, but I know the expo installs of these libraries have the same issue on iOS.

### Expected behaviour
Alternating swipes left and right should listen and respond properly to their respective actions. 

### What have you tried
I initially was just using a drawerLayout on the main screen of a stackNavigator which was in the tabNavigator, but I switched to drawerNavigator to hopefully fix the issue but it is the same. I have also tried the current running expo version of react-navigation-tabs (1.1.4), the suggested fix in react-navigation/tabs#134 (2.0 alpha), and the current release (2.3) all without any improvement.

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        | not sure where to check.
| react-native          |  0.59.8
| react-navigation-tabs | 1.1.4, 2.0 alpha, 2.3
| node                  | 10.16.1
| npm or yarn           | 6.9.0 (node) 1.17.3 (yarn)
